extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -0,0 +1,644 @@
+			await this.loadblobStorageDropdown();
+		});
+
+		const storageAccountLabel = view.modelBuilder.text()
+			.withProps({
+				value: constants.DATABASE_BACKUP_BLOB_STORAGE_ACCOUNT_LABEL,
+				requiredIndicator: true,
+			}).component();
+		this._blobContainerStorageAccountDropdown = view.modelBuilder.dropDown()
+			.withProps({
+				required: true
+			}).component();
+		this._blobContainerStorageAccountDropdown.onValueChanged(async (value) => {
+			this._blob.storageAccountId = (this._blobContainerStorageAccountDropdown.value as azdata.CategoryValue).name;
+			await this.loadBlobContainerDropdown();
+		});
+
+		const containerLabel = view.modelBuilder.text().withProps({
+			value: constants.DATABASE_BACKUP_BLOB_STORAGE_ACCOUNT_CONTAINER_LABEL,
+			requiredIndicator: true,
+		}).component();
+		this._blobContainerBlobDropdown = view.modelBuilder.dropDown()
+			.withProps({
+				required: true
+			}).component();
+		this._blobContainerBlobDropdown.onValueChanged((value) => {
+			this._blob.containerId = (this._blobContainerBlobDropdown.value as azdata.CategoryValue).name;
+		});
+
+		const flexContainer = view.modelBuilder.flexContainer()
+			.withItems(
+				[
+					subscriptionLabel,
+					this._blobContainerSubscriptionDropdown,
+					storageAccountLabel,
+					this._blobContainerStorageAccountDropdown,
+					containerLabel,
+					this._blobContainerBlobDropdown
+				]
+			).withLayout({
+				flexFlow: 'column'
+			}).component();
+
+		return flexContainer;
+	}
+
+	private createNetworkShareContainer(view: azdata.ModelView): azdata.FlexContainer {
+		const networkShareHelpText = view.modelBuilder.text()
+			.withProps({
+				value: constants.DATABASE_BACKUP_NC_NETWORK_SHARE_HELP_TEXT,
+			}).component();
+
+		const networkShareLocationLabel = view.modelBuilder.text()
+			.withProps({
+				value: constants.DATABASE_BACKUP_NETWORK_SHARE_LOCATION_LABEL,
+				requiredIndicator: true,
+			}).component();
+		this._networkShareLocationText = view.modelBuilder.inputBox()
+			.withProps({
+				placeHolder: '\\\\Servername.domainname.com\\Backupfolder',
+				required: true
+			})
+			.withValidation((component) => {
+				if (component.value) {
+					if (!/^(\\)(\\[\w\.-_]+){2,}(\\?)$/.test(component.value)) {
+						return false;
+					}
+				}
+				return true;
+			}).component();
+		this._networkShareLocationText.onTextChanged((value) => {
+			this._networkShare.networkShareLocation = value;
+		});
+
+		const windowsUserAccountLabel = view.modelBuilder.text()
+			.withProps({
+				value: constants.DATABASE_BACKUP_NETWORK_SHARE_WINDOWS_USER_LABEL,
+				requiredIndicator: true,
+			}).component();
+		this._windowsUserAccountText = view.modelBuilder.inputBox()
+			.withProps({
+				placeHolder: 'Domain\\username',
+				required: true
+			})
+			.withValidation((component) => {
+				if (component.value) {
+					if (!/^[a-zA-Z][a-zA-Z0-9\-\.]{0,61}[a-zA-Z]\\\w[\w\.\- ]*$/.test(component.value)) {
+						return false;
```

Comment:
You should probably have a descriptive error message here for when this validation fails